js-plugins/mcp/src/index.ts

```
@@ -0,0 +1,79 @@
+import { existsSync } from 'fs';
+import { join } from 'path';
+
+
+export default function farmPluginMcp(options: FarmMcpOptions): JsPlugin {
+  const {
+    mcpPath = '/__mcp',
+    updateCursorMcpJson = true,
+    updateVSCodeMcpJson = true,
+    printUrl = true,
+  } = options
+  const cursorMcpOptions = typeof updateCursorMcpJson == 'boolean'
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)